server/game/cards/07-GR/PhantomDrummernaut.js

```diff
@@ -0,0 +1,31 @@
+const Card = require('../../Card.js');
+
+class PhantomDrummernaut extends Card {
+    // Play/After Fight: Return a creature from your discard pile to
+    // your hand.
+    //
+    // Destroyed: If you are haunted, archive Encounter Golem.
```

Comment:
Should say `Destroyed: If you are haunted, archive Phantom Drummernaut.`

Reply:
Done.